src/Apps/MyCollection/Routes/EditArtwork/MyCollectionArtworkForm.tsx

```diff
@@ -116,6 +124,11 @@ export const MyCollectionArtworkForm: React.FC<MyCollectionArtworkFormProps> = (
         artworkLocation: values.artworkLocation,
       })
 
+      // Adding tracking after a successfully adding an artwork
+      if (!isEditing) {
+        trackEvent(tracks.saveCollectedArtwork())
+      }
```

Comment:
**suggestion:** I could imagine that the most important info here is that we don't track if the user saves an artwork after editing. What do you think about emphasizing this in the comment? For example like this:

```suggestion
      //  Track saving only when the user adds a new artwork.
      if (!isEditing) {
        trackEvent(tracks.saveCollectedArtwork())
      }
```